bundles/org.openhab.binding.digiplex/src/main/java/org/openhab/binding/digiplex/internal/handler/DigiplexBridgeHandler.java

```diff
@@ -33,11 +33,7 @@
 import org.eclipse.jdt.annotation.Nullable;
 import org.openhab.binding.digiplex.internal.DigiplexBindingConstants;
 import org.openhab.binding.digiplex.internal.DigiplexBridgeConfiguration;
-import org.openhab.binding.digiplex.internal.communication.CommunicationStatus;
-import org.openhab.binding.digiplex.internal.communication.DigiplexMessageHandler;
-import org.openhab.binding.digiplex.internal.communication.DigiplexRequest;
-import org.openhab.binding.digiplex.internal.communication.DigiplexResponse;
-import org.openhab.binding.digiplex.internal.communication.DigiplexResponseResolver;
+import org.openhab.binding.digiplex.internal.communication.*;
```

Comment:
Wildcard imports should only be used for static imports. This also generates a SAT warning.

Reply:
Ah, new Intellij setup and 'optimize imports' settings... good catch!

Fixed